frontend/src/components/annotation/annotation_ui_factory.vue

```diff
@@ -589,14 +594,25 @@ export default Vue.extend({
         if (this.enabled_edit_schema == true) {
             this.edit_ui_schema()
         }
+        this.hotkey_listener = HotkeyListener.getInstance()
+
+        this.hotkey_listener.addFilter(() => {
+          return !this.$store.state.user.is_typing_or_menu_open
+        })
+
+        // NOTE: setting scope to image here for now so event handlers get activated
+        // for annotation_core.vue. Move this logic to a function that fires when
+        // selected file changes
+        this.hotkey_listener.addScope('image')
     },
     beforeDestroy() {
         this.hotkey_manager.deactivate()
         window.removeEventListener(
             "resize",
             this.update_window_size_from_listener
         );
-
+        this.hotkey_listener.clear()
+        this.hotkey_listener.restoreDefaultFilters()
```

Comment:
`hotkey_listener.restoreDefaultFilters` remove all the filters that we added via `hotkey_listener.addFilter` and restores default filtering logic provided by hotkey-js library. The default logic I'm talking about prevents hotkeys from firing if user is focused on some input filed (input, textarea, etc) or element that has `contenteditable` ([docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable)) set.